src/DatadogLoggingService.js

```diff
@@ -28,13 +28,25 @@ class DatadogLoggingService extends NewRelicLoggingService {
   }
 
   initialize() {
+    const requiredDatadogConfig = [
+      process.env.DATADOG_APPLICATION_ID,
+      process.env.DATADOG_CLIENT_TOKEN,
+    ];
+    const hasRequiredDatadogConfig = requiredDatadogConfig.every(value => !!value);
+
+    // Do not attempt to initialize Datadog if required config settings are not supplied.
+    if (!hasRequiredDatadogConfig) {
+      return;
+    }
+
+    const datadogVersion = process.env.DATADOG_VERSION || process.env.APP_VERSION || '1.0.0';
     datadogRum.init({
-      applicationId: process.env.DATADOG_APPLICATION_ID,
-      clientToken: process.env.DATADOG_CLIENT_TOKEN,
-      site: process.env.DATADOG_SITE,
-      service: process.env.DATADOG_SERVICE,
-      env: process.env.DATADOG_ENV,
-      version: process.env.DATADOG_VERSION,
+      applicationId: process.env.DATADOG_APPLICATION_ID || '',
```

Comment:
nit: given the required config settings check is implemented above, the `applicationId` and `clientToken` properties probably don't need to fallback to an empty string since they should be expected to have a value at this point.

The other fallbacks to empty strings for the other properties still makes sense, though :)